src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -855,12 +1256,22 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
                         ms2GroupExists,
                         includeSetNamesLines,
                         _mainwindow->mavenParameters);
-  QList<shared_ptr<PeakGroup>> selectedGroups = getSelectedGroups();
+  QList<shared_ptr<PeakGroup>> selectedGroups;
+  
+  if (static_cast<int>(peakTableSelection) == 10)
+    selectedGroups = getDisplayedGroups();
+  else
+    selectedGroups = getSelectedGroups();
```

Comment:
Does that `10` mean something special here? Please use a constant or at least add a comment.